src/entities-and-resolution.rst

```diff
@@ -1006,36 +1010,113 @@ A :t:`use import` brings :t:`entities <entity>` :t:`in scope` within the
 A :t:`simple path prefix` is the leading :t:`simple path` of a :t:`glob import`
 or a :t:`nesting import`.
 
+
+.. TODO simple path prefix, path prefix and import path prefix, need better terms
+
+:dp:`fls_WAA4WmohGu6T`
+An :dt:`import path prefix` is the fully constructed :t:`path` prefix of a
+:t:`use import`. An :t:`import path prefix` for a given
+:t:`simple import` or :t:`glob import` is constructed as follows:
+
+#. :dp:`fls_IPYvldMqduf4`
+   Start the :t:`import path prefix` as follows:
+
+   * :dp:`fls_MOXId37fcNPY`
+     If the :t:`use import` is a :t:`simple import` then start with the
+     :t:`[path segment]s` of the :t:`simple import`'s :t:`simple path`
+     :t:`path prefix`.
+
+   * :dp:`fls_2UyFcB6Our1v`
+     If the :t:`use import` is a :t:`glob import` then start with the
+     :t:`[path segment]s` of the :t:`glob import`'s :t:`simple path prefix`.
+
+   * :dp:`fls_irdKqoYzBM0M`
+     If the :t:`use import` is a :t:`nesting import` then start with the
+     :t:`[path segment]s` of the :t:`nesting import`'s :t:`simple path prefix`.
+
+#. :dp:`fls_gAWsqibl4GLq`
+   Then if the current :t:`use import` is the child of a :t:`nesting import`,
+   prepend the :t:`nesting import`'s :t:`simple path prefix` to the
+   :t:`import path prefix`. Repeat this step with the :t:`nesting import` as
+   the current :t:`use import`.
+
+:dp:`fls_2bkcn83smy2y`
+A :t:`simple import` is a :t:`use import` that brings all :t:`entities <entity>`
+it refers to into scope, optionally with a different
+:t:`name` than they are declared with by using a :t:`renaming`.
+
 :dp:`fls_v3a6y2ze44v2`
 A :t:`glob import` is a :t:`use import` that brings all :t:`entities <entity>`
-with :t:`public visibility` prefixed by its :t:`simple path prefix` into
-:t:`scope`.
+exported by the :t:`module` or :t:`enum` its :t:`import path prefix` resolves to
+into :t:`scope`.
```

Comment:
This sentence is quite hard to understand: "
"A glob import is a use import that brings all entities exported by the module or enum its import path prefix resolves to into scope."
Do you mean it brings all modules pointed to by the import path prefix into scope?

Reply:
it brings all entities into scope that are exported in the module or enum that the path prefix resolves to, that is
```
foo::bar::*
```
imports everything from the `foo::bar` module (or enum)

Reply:
If it is clear to you, then that is good enough.